css/home.css

```diff
@@ -192,7 +192,7 @@
     height: 198px;
   }
 
-  .homepage-hero-anchor {
+  .homepage-hero__anchor {
```

Comment:
wasnt this merged in your last pr?

Reply:
yep

Reply:
I dont know why take change here, but that it's the same

Reply:
Please make a pull from develop to ensure you have all up to date.

Reply:
rdy